src/util/util_env.cpp

```diff
@@ -184,6 +184,7 @@ namespace dxvk::env {
     if (overrideStr == #arch) \
         override = NV_GPU_ARCHITECTURE_##arch;
             CHECK_ARCH(GK100)
+            CHECK_ARCH(GM000)
```

Comment:
We should also add this to the readme at https://github.com/jp7677/dxvk-nvapi/blob/master/README.md#tweaks-debugging-and-troubleshooting